scripts/configure-installer-tests-cluster.sh

```diff
@@ -20,6 +20,9 @@ export KUBECONFIG=$ORIGINAL_KUBECONFIG
 # List of users to create
 USERS="developer odonoprojectattemptscreate odosingleprojectattemptscreate odologinnoproject odologinsingleproject1"
 
+# list of namespace to create
+NAMESPACE="openjdk-11-rhel8 nodejs-12-rhel7"
```

Comment:
`NAMESPACE` is a little bit confusing name :-( Could we do something like `IMAGE_NAMESPACES` or `IMAGE_TEST_NAMESPACES`?